ui-particles-angular/projects/ui-particles-angular/src/lib/gio-monaco-editor/gio-monaco-editor.component.ts

```diff
@@ -37,8 +37,8 @@ import { GioLanguageJsonService } from './services/gio-language-json.service';
 import { GioMonacoEditorService } from './services/gio-monaco-editor.service';
 
 export type MonacoEditorLanguageConfig = {
-  language: 'json';
-  schemas: { uri: string; schema: unknown }[];
+  language: 'json' | 'markdown';
```

Comment:
WDYT to change to this : 
```
{
  language: 'json';
  schemas: { uri: string; schema: unknown }[];
} | {
  language: 'markdown';
}
```
to allow to have specific config for each language

Reply:
Yes much better ! Done.